internal/converter/util/well_known.go

```diff
@@ -188,10 +188,6 @@ func NewGoogleAny() *IDSchema {
 		Type:   []string{"string"},
 		Format: "binary",
 	}))
-	props.Set("debug", base.CreateSchemaProxy(&base.Schema{
-		Type:                 []string{"object"},
-		AdditionalProperties: &base.DynamicValue[*base.SchemaProxy, bool]{N: 1, B: true},
-	}))
```

Comment:
This isn't a general feature of all `google.protobuf.Any` with ConnectRPC. It's just with errors from ConnectRPC